vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```
@@ -138,7 +138,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>winsqlite3.lib;ChakraRT.lib;dxguid.lib;dloadhelper.lib;OneCoreUap_apiset.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+      <DelayLoadDLLs>api-ms-win-core-file-l1-2-0.dll;ext-ms-win-uiacore-l1-1-1.dll;api-ms-win-core-windowserrorreporting-l1-1-0.dll;chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

(nit)

For readability, can we do one item per line?
i.e.
```xml
<DelayLoadDLLs>
  api-ms-win-core-file-l1-2-0.dll;
  ext-ms-win-uiacore-l1-1-1.dll;
  api-ms-win-core-windowserrorreporting-l1-1-0.dll;
  chakra.dll;
  winsqlite3.dll;
  %(DelayLoadDLLs)
</DelayLoadDLLs>
```

---

will do : ) FYI you can use the _suggestion_ format to suggest a change; helpful when it's simple like this :)
e.g.:
```
    ```suggestion
    foo
    ```
```